dig_test.go

```diff
@@ -2990,3 +2990,125 @@ func TestUnexportedFieldsFailures(t *testing.T) {
 			`bad argument 1: invalid value "foo" for "ignore-unexported" tag on field In: strconv.ParseBool: parsing "foo": invalid syntax`)
 	})
 }
+
+func TestProvideInfoOption(t *testing.T) {
+	t.Parallel()
+	t.Run("two outputs", func(t *testing.T) {
+		type type1 struct{}
+		type type2 struct{}
+		ctor := func() (*type1, *type2) {
+			return &type1{}, &type2{}
+		}
+
+		c := New()
+		info := ProvideInfo{}
```

Comment:
minor stylistic nit: use var form for zero value.

https://github.com/uber-go/guide/blob/master/style.md#use-var-for-zero-value-structs
```suggestion
		var info ProvideInfo
```